content/intermediate/templates/nested-stacks/index.md

```diff
@@ -426,23 +496,61 @@ Before you can deploy the updated nested stack, you must update the templates in
+  cd cfn101-workshop/code/workspace/nested-stacks
+  :::
+1. Run the below command to copy the `iam.yaml`, `vpc.yaml` and `ec2.yaml` to S3 bucket. Replace the `bucket-name` with the bucket name from previous step.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws s3 cp iam.yaml s3://{bucket-name}
+  aws s3 cp vpc.yaml s3://{bucket-name}
+  aws s3 cp ec2.yaml s3://{bucket-name}
+  :::
+::::
+::::tab{id="local" label="Local development"}
 1. Navigate to your S3 bucket in the console and select it.
 1. Click on **Upload** button -> **Add files**.
 1. Locate the `vpc.yaml`, `iam.yaml` and `ec2.yaml` files and select them.
 1. Click **Upload** button to upload the file.
+::::
+:::::
 
 ##### 9. Deploy EC2 Nested Stack
 
 Update the previously created nested stack with a new template.
 
+:::::tabs{variant="container"}
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace/nested-stacks`:
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  cd cfn101-workshop/code/workspace/nested-stacks
+  :::
+1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Replace the `ParameterValue` **bucketName** with the value you have written down in [Prepare S3 bucket](#2-prepare-s3-bucket) section. Replace the `ParameterValue` **AZ1** and  **AZ2** with the value of any 2 AZ's from your vpc.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation update-stack --stack-name cfn-workshop-nested-stacks --template-body file://main.yaml --parameters ParameterKey="S3BucketName",ParameterValue="bucketName" ParameterKey="AvailabilityZones",ParameterValue=AZ1\\,AZ2 --capabilities CAPABILITY_NAMED_IAM
+  :::
+1. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
+  :::code{language=shell showLineNumbers=false showCopyAction=false}
+  "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-nested-stacks/739fafa0-e4d7-11ed-a000-12d9009553ff"
+  :::
+1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** console in a new tab and check if the stack status is **CREAUPDATETE_COMPLETE**.
+::::
+
+::::tab{id="local" label="Local development"}
 1. Navigate to Cloudformation service in the AWS console.
 1. Select the **root** stack (it is the one without the **nested** tag associated).
 1. In the top right corner click on **Update**.
 1. In **Prepare template**, choose **Replace current template**.
 1. In **Template source**, choose **Upload a template file**.
 1. Click on **Choose file** button and navigate to your workshop directory.
 1. Select the `main.yaml` template file and click **Next**.
-1. Follow the wizard, acknowledge IAM capabilities and click on **Update stack**.
+1. Follow the wizard, acknowledge IAM capabilities and click on **Submit**.
+
+::::
+:::::
 
 #### 7. Test the deployment
```

Comment:
Note in the Session Manager lab we don't describe how to do this, we use an outdated animated gif. I suggest we just link to the console on line 571
```
Verify that you can log in to the instance via **[Session Manager](https://us-east-1.console.aws.amazon.com/systems-manager/session-manager/sessions?region=us-east-1)** and select your region.
```

Reply:
Nice catch, yeah lets do that...